web/api/mocks/data/steps.json

```diff
@@ -1,27 +1,29 @@
 [
   {
     "id": "step-1-flow-1",
-    "name": "Flow 01 Ingest Step",
+    "name": "Flow01IngestStep",
     "description": "My Step 1 description",
     "type": "ingest",
     "sourceDatabase": "",
     "targetDatabase": "staging",
```

Comment:
https://project.marklogic.com/jira/browse/DHFPROD-2096?focusedCommentId=153360&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-153360
targetDatabase and sourceDatabase need to be nested in options